app/views/videos/index.html.erb

```diff
@@ -0,0 +1,9 @@
+<%= form_tag videos_path, :method => 'get', :class => 'form-search', :id => 'video-search', :autocomplete => 'off' do %>
```

Comment:
I turned off autocomplete because it was obscuring the results, but we may want to toss it back in eventually.